Description: Link only Foundation library against -ldl and -lrt on Linux and FreeBSD.
Origin: vendor
Forwarded: http://sourceforge.net/tracker/?func=detail&atid=725711&aid=3470675&group_id=132964
Author: Cristian Greco <cristian@debian.org>

--- a/build/config/Linux
+++ b/build/config/Linux
@@ -68,4 +68,4 @@
 #
 # System Specific Libraries
 #
-SYSLIBS  = -lpthread -ldl -lrt
+SYSLIBS  = -lpthread
--- a/Foundation/Makefile
+++ b/Foundation/Makefile
@@ -57,6 +57,10 @@
 	objects += SyslogChannel
 endif
 
+ifneq ($(or $(findstring Linux, $(POCO_CONFIG)), $(findstring FreeBSD, $(POCO_CONFIG))), )
+	SYSLIBS += -ldl -lrt
+endif
+
 target         = PocoFoundation
 target_version = $(LIBVERSION)
 target_libs    =
